src/protocolsupport/protocol/typeremapper/chunk/ChunkTransformerPE.java

```diff
@@ -62,8 +70,8 @@ public void writeLegacyData(ByteBuf chunkdata) {
 		}
 	}
 
-	protected int getBlockState(int section, BlockStorageReader storage, int x, int y, int z) {
-		return getBlockState(section, storage, (y << 8) | (z << 4) | (x));
+	private int getPcIndex(int x, int y, int z) {
```

Comment:
We use protected everywhere we can to avoid generation of accessor methods. Also this method can now be static.